var/spack/repos/builtin/packages/r-blob/package.py

```diff
@@ -15,9 +15,9 @@ class RBlob(RPackage):
     for use as a column in data frame."""
 
     homepage = "https://cloud.r-project.org/package=blob"
```

Comment:
You can remove this line and let the `cran` attribute handle it, or use: https://blob.tidyverse.org

Reply:
Done.